package.json

```diff
@@ -26,22 +26,23 @@
     "@fuel-ui/react": "^0.16.1",
     "@fuel-wallet/sdk": "^0.10.0",
     "@mdx-js/react": "^2.3.0",
-    "@next/mdx": "^13.4.8",
+    "@next/mdx": "^13.4.9",
     "acorn": "^8.10.0",
     "acorn-loose": "^8.3.0",
     "acorn-walk": "^8.2.0",
+    "cookies-next": "^2.1.2",
     "dotenv": "^16.3.1",
     "framer-motion": "^10.12.18",
     "fuels": "0.45.0",
-    "globby": "^13.2.1",
+    "globby": "^13.2.2",
     "gray-matter": "^4.0.3",
     "hast-util-heading-rank": "^2.1.1",
     "hast-util-to-string": "^2.0.0",
-    "next": "13.4.8",
+    "next": "13.4.9",
     "next-mdx-remote": "^4.4.1",
     "node-html-markdown": "^1.3.0",
     "plyr-react": "^5.3.0",
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
```

Comment:
I had changed this back before because it causes an error with the version of the wallet docs we are using.